content/rust/content/structs/structs.md

```diff
@@ -0,0 +1,39 @@
+---
+Title: 'Structs'
+Description: 'Rust structs are custom data structures used to group related data fields together. They define the blueprint for creating instances with named fields, enabling organized and type-safe data storage.'
+Subjects:
+  - 'Data Modeling'
+  - 'Web Development'
+  - 'Blockchain and Cryptocurrency'
+  - 'Graphics and Computer Vision'
```

Comment:
```suggestion
  - 'Data Science'
  - 'Web Development'
 ```
Subjects should be two entries from [this list](https://github.com/Codecademy/docs/blob/main/documentation/subjects.md)